gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -293,7 +321,9 @@ public GoogleCloudStorageImpl(GoogleCloudStorageOptions options, Storage storage
       }
     }
 
-    this.storageRequestAuthorizer = initializeStorageRequestAuthorizer(storageOptions);
+    this.storageRequestAuthorizer = initializeStorageRequestAuthorizer(this.storageOptions);
+    this.refreshAccessTokenPerRequestEnabled = this.storageOptions.getRefreshAccessTokenPerRequestEnabled();
```

Comment:
Don't you just have this boolean value in the Storage Options?

Reply:
yeah, I can pull it from the options at anytime. Let me do that change